packages/lodestar/src/chain/chain.ts

```diff
@@ -226,24 +237,9 @@ export class BeaconChain extends (EventEmitter as {new (): ChainEventEmitter}) i
       `Initializing beacon chain with state root ${toHexString(stateRoot)}` +
         ` and block root ${toHexString(blockRoot)}, number of validator: ${genesisState.validators.length}`
     );
-    const justifiedFinalizedCheckpoint = {
-      root: blockRoot,
-      epoch: computeEpochAtSlot(this.config, genesisBlock.slot),
-    };
-    this.forkChoice.addBlock({
-      slot: genesisBlock.slot,
-      blockRoot: blockRoot,
-      stateRoot: stateRoot,
-      parentRoot: Buffer.alloc(32),
-      justifiedCheckpoint: justifiedFinalizedCheckpoint,
-      finalizedCheckpoint: justifiedFinalizedCheckpoint,
-    });
-    const epochCtx = new EpochContext(this.config);
-    epochCtx.loadState(genesisState);
-    await this.db.stateCache.add({state: genesisState, epochCtx});
```

Comment:
where did this part of code go?

Reply:
We don't need this since we're doing the same in chain.start